docs/userGuide/usingComponents.md

```diff
@@ -205,4 +205,47 @@ If you want to include a keyword without rendering it to the page, add `d-none`
 This is good for catching <span class="keyword">regressions</span>
 ```
 
+# Using Tags
+
+You can use tags allow you to selectively display or hide certain elements on the compiled static site.
+
+A tag is an attribute attached to any HTML element. Tags are specified by the `tags` attribute, For example:
+
+```html
+# Welcome to my site
+
+<p tags="language--english">Hello</p>
+<div tags="language--french">Bonjour</div>
+<span tags="language--spanish">Hola</span>
+```
+
+Tagged elements to include are specified in the `site.json` file, under the `tags` option:
+
+```
+{
+  ...
+  "tags": ["language--english"]
+}
+```
+
+When compiled, only elements that match tags specified in the `site.json` files will be displayed. All other tagged elements will be hidden. In this case, only the element with the `language--english` tag will be displayed. This is helpful when creating multiple versions of a page without creating multiple copies.
+
+Elements can also have multiple tags, separated by a space. An element will be displayed if any tag matches the one in `site.json`.

```

Comment:
~~Elements can also have~~ **You can also use multiple tags in a single HTML element. Specify each tag in the `tag` attribute ** separated by a space. An element will be displayed if any **of the tags** matches the one in `site.json`.